test/format/config.c

```diff
@@ -272,7 +272,7 @@ config_table(TABLE *table, void *arg)
           config_explicit(NULL, "runs.in_memory")) {
             WARN("limiting table%" PRIu32
                  ".runs.rows to 1,000,000 as runs.in_memory has been automatically enabled",
-              table->id)
+              table->id);
```

Comment:
The `WARN` macro is defined in `format.h` and it includes the trailing semicolon. So the old call was correct. If you want to force the use of the semicolon like here, then remove it from the macro too. I think this looks/reads better so I'm in favor of removing it from the `format.h` macro.

Reply:
fixed